docs/maintainers/sass-compilation.md

```diff
@@ -0,0 +1,55 @@
+# play-frontend-hmrc: SASS compilation
+
+This library manages SASS compilation for you. However, should you wish to add your own for any reason using 
+`sbt-sassify`, please read the steps below.
+
+1.  Import the required styles in your `app/assets/stylesheets/application.scss` file:
+    ```scss
+    @import "lib/govuk-frontend/govuk/base";
+    
+    // Add your custom styles here
+    ```
+Ensure [sbt-sassify](https://github.com/irundaia/sbt-sassify) is added to plugins.sbt
+
+1. Ensure you have the correct routing for all other static assets including the compiled Javascript and images provided
+   by the hmrc-frontend webjar:
+    ```
+    GET        /assets/*file                        controllers.Assets.versioned(path = "/public", file: Asset)
+    ```
+   
+1. You will need to create a custom `headBlock` template to include your styles. The content of the file might look 
+   something like this:
+```html
+@this()
+
+@()
+<!--[if lte IE 8]><link href='@controllers.routes.Assets.versioned("stylesheets/application-ie-8.css")' rel="stylesheet" type="text/css" /><![endif]-->
```

Comment:
I think the conditional stuff for IE8 can be removed. We're not generating an IE8 specific stylesheet here anyway.